ci-operator/step-registry/sandboxed-containers-operator/post/sandboxed-containers-operator-post-commands.sh

```
@@ -21,9 +23,91 @@ cleanup_kata() {
+			--region "${aws_region}" \
+			--instance-ids ${podvm_instances} 2>/dev/null || true
+
+		echo "Waiting for PodVM instances to terminate (up to 5 minutes)..."
+		# aws ec2 wait polls every 15s, times out after 40 attempts (10 min).
+		# We use a shorter manual loop to cap at ~5 minutes.
+		local retries=0
+		while [[ ${retries} -lt 20 ]]; do
+			local still_running
+			still_running=$(aws ec2 describe-instances \
+				--region "${aws_region}" \
+				--instance-ids ${podvm_instances} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)